tests/Integration/Transport/HttpTest.php

```diff
@@ -95,4 +97,56 @@ public function testSilentlySendTraces()
 
         $this->assertSame('', $output);
     }
+
+    public function testSendsMetaHeaders()
+    {
+        $process = new Process("exec php -S localhost:8500 -t " . __DIR__ . "/request_replayer");
+        $process->start();
+        usleep(100000);
+
+        $httpTransport = new Http(new Json(), null, [
+            'endpoint' => 'http://localhost:8500/test-trace',
+        ]);
+        $tracer = new Tracer($httpTransport);
+
+        $span = $tracer->startSpan('test');
+        $span->finish();
+
+        $traces = [[$span]];
+        $httpTransport->send($traces);
+
+        $traceRequest = json_decode(file_get_contents('http://localhost:8500/replay'), true);
+
+        $this->assertEquals('php', $traceRequest['headers']['Datadog-Meta-Lang']);
+        $this->assertEquals(\PHP_VERSION, $traceRequest['headers']['Datadog-Meta-Lang-Version']);
+        $this->assertEquals(\PHP_SAPI, $traceRequest['headers']['Datadog-Meta-Lang-Interpreter']);
+        $this->assertEquals(Version\VERSION, $traceRequest['headers']['Datadog-Meta-Tracer-Version']);
+
+        $process->stop(0);
+    }
+
+    public function testSetHeader()
+    {
+        $process = new Process("exec php -S localhost:8500 -t " . __DIR__ . "/request_replayer");
+        $process->start();
+        usleep(100000);
+
+        $httpTransport = new Http(new Json(), null, [
+            'endpoint' => 'http://localhost:8500/test-trace',
```

Comment:
I feel like this URL hostname + port should also be abstracted away, so that all those magic strings and numbers would live in only one place.